docs/source/guides/functions.rst

```diff
@@ -556,7 +561,7 @@ This is a test example of :func:`functions.relu` function
            x = Variable(np.random.randn(3, 2).astype(np.float32))
            y = F.relu(x)
            y.grad = np.random.randn(3, 2).astype(np.float32)
-           y.backward()
+           y.backward(retain_grad=True)
```

Comment:
Currently `retain_grad=True` is needed due to https://github.com/chainer/chainer/pull/6348